packages/x-scheduler/src/primitives/use-event-calendar/tests/core.EventCalendarInstance.test.ts

```diff
@@ -0,0 +1,250 @@
+import { getAdapter } from './../../utils/adapter/getAdapter';
+import {
+  DEFAULT_PREFERENCES,
+  DEFAULT_PREFERENCES_MENU_CONFIG,
+  DEFAULT_VIEW,
+  DEFAULT_VIEWS,
+  EventCalendarInstance,
+} from '../EventCalendarInstance';
+import { buildEvent, getIds } from './utils';
+import { CalendarView } from '../../models';
+
+const adapter = getAdapter();
+const DEFAULT_PARAMS = { events: [] };
+
+describe('Core - EventCalendarInstance', () => {
+  describe('create', () => {
+    it('initializes default state', () => {
```

Comment:
```suggestion
    it('should initialize default state', () => {
```

x other tests

We have the convention on almost every test in the company, let's keep it even if it's super subjective 👍